CIPs/CIP-59/CIP-59.md

```diff
@@ -12,58 +12,81 @@ edited: 2020-09-24
 
 ## Simple Summary
 
-Specification for how to encode a Document Id (DocID)
+Specification for how to encode a Stream Id (StreamID)
 
 
 ## Abstract
-A DocID is composed of a docid-multicodec, a doctype varint, and a CID.
+A StreamID is composed of a streamid-multicodec, a stream type varint, and a CID.
 
 
 ## Motivation
-A specific encoding for DocIDs allows us to distinguish them from CIDs as well as provide more information about the given document. 
+A specific encoding for StreamIDs allows us to distinguish them from CIDs as well as provide more information about the given stream. CommitIDs are a subset of a StreamID which refers to a specific commit in a stream.
 
 
 ## Specification
-DocIDs are defined as:
+#### StreamID
+
+StreamIDs are defined as:
 
 ```html
-<docid> ::= <multibase-prefix><multicodec-docid><doctype><genesis-cid-bytes>
+<streamid> ::= <multibase-prefix><multicodec-streamid><stream-type><genesis-cid-bytes>
 
 # e.g. using CIDv1
-<docid> ::= <multibase-prefix><multicodec-docid><doctype><multicodec-cidv1><multicodec-content-type><multihash-content-address>
+<streamid> ::= <multibase-prefix><multicodec-streamid><stream-type><multicodec-cidv1><multicodec-content-type><multihash-content-address>
 ```
 
 Where
 
-- `<multibase-prefix>` is a [multibase](https://github.com/multiformats/multibase) code (1 or 2 bytes), to ease encoding DocIDs into various bases. **NOTE:** *Binary* (not text-based) protocols and formats may omit the multibase prefix when the encoding is unambiguous.
-- `<multicodec-docid>` is a [multicodec](https://github.com/multiformats/multicodec) used to indicate that it's a DocID.
-- `<doctype>` is a [varint](https://github.com/multiformats/unsigned-varint) representing the doctype of the document.
+- `<multibase-prefix>` is a [multibase](https://github.com/multiformats/multibase) code (1 or 2 bytes), to ease encoding StreamIDs into various bases. **NOTE:** *Binary* (not text-based) protocols and formats may omit the multibase prefix when the encoding is unambiguous.
+- `<multicodec-streamid>` is a [multicodec](https://github.com/multiformats/multicodec) used to indicate that it's a StreamID.
```

Comment:
I guess we'll need to make a PR to update https://github.com/multiformats/multicodec/blob/master/table.csv as well